src/DiadocHttpApi.CloudCrypt.cs

```diff
@@ -65,6 +65,15 @@ public AsyncMethodResult DssSign(string authToken, string boxId, DssSignRequest
 			return PerformHttpRequest<AsyncMethodResult>(authToken, "POST", queryString.BuildPathAndQuery(), Serialize(request));
 		}
 
+		[Obsolete("Currently unavailable. Use DssSign")]
+		public AsyncMethodResult DssSignV2(string authToken, string boxId, DssSignRequestV2 request)
+		{
+			var queryString = new PathAndQueryBuilder("V2/DssSign");
```

Comment:
Для однообразия с остальными методами

```suggestion
			var queryString = new PathAndQueryBuilder("/V2/DssSign");
```

Reply:
Поправил